runtime/Ansible.py

```diff
@@ -403,6 +403,21 @@ def package_confirm(confirm):
                         str(e),
                         event=BAD_EVENTS.TCP_ERROR,
                         printStackTrace=True))
+        def package_timestamps(timestamp):
```

Comment:
Sorry if this wasn't clear but we should change the argument name from timestamp to timestamps and change the rest of the function to reflect this change